tokenserver/__init__.py

```diff
@@ -28,9 +28,10 @@
 
 def includeme(config):
     settings = config.registry.settings
+    """
     if settings.get('tokenserver.monkey_patch_gevent', True):
         monkey_patch_gevent()
-
+    """
```

Comment:
I'm kind of inclined not to have folks use what may be a dead version of python. Pypy has stated that they're going to support the Python2 syntax, so I'd feel a bit more comfortable punting folks over that way since low level security bugs would have a better chance to get fixed. 

And thanks for spotting that commented out monkeypatch. Missed that one while cleaning up.